chart/templates/grafana-datasources-sec.yaml

```
@@ -1,6 +1,6 @@
 {{- $grafana := index .Values "kube-prometheus-stack" "grafana" -}}
 {{ if and $grafana.enabled $grafana.sidecar.datasources.enabled -}}
-{{- $tsdbExternalEnabled := .Values.timescaledbExternal.enabled -}}
+{{- $isDBURI := ternary true false (ne .Values.promscale.connection.uri "")}}
```

Why `ternary` is needed here? Why not just:
```suggestion
{{- $isDBURI := (ne .Values.promscale.connection.uri "")}}
```